browse/templates/abs/abs.html

```diff
@@ -18,7 +18,11 @@
 {% block body_id %}{% endblock %}
 
 {% block header_h1 %}
-  <div class="header-breadcrumbs is-hidden-mobile"><a href="/">{{ config['BROWSE_SITE_LABEL'] }}</a> &gt; <a href="{{ url_for('.list_articles',context=abs_meta.primary_archive.id, subcontext='recent') }}">{{ abs_meta.primary_archive.id }}</a> &gt; arXiv:{{ requested_id }}</div>
+  <div class="header-breadcrumbs is-hidden-mobile">
+    {% if config['BROWSE_SITE_HOST'] != None %}<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="beta-exclamation"><path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zm-248 50c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"/></svg>{% endif %}
+    <a href="/">{{ config['BROWSE_SITE_LABEL'] }}</a> &gt; <a href="{{ url_for('.list_articles',context=abs_meta.primary_archive.id, subcontext='recent') }}">{{ abs_meta.primary_archive.id }}</a> &gt; arXiv:{{ requested_id }}
+    {% if config['BROWSE_SITE_HOST'] != None %}<a href="https://arxiv.org/" class="return-link-abs">(go to arxiv.org)</a>{% endif %}
```

Comment:
These are the lines in question to review.

Reply:
Will it always be the case that setting BROWSE_SITE_HOST will indicate the host is a development box? Would this be set in a similar way for mirrors (once we update mirrors to use new browse)? May not be a factor now.

Reply:
I've made a small change here to check for "dev" or "beta" in `BROWSE_SITE_LABEL`.

@SBBCornell if you'd like to test this locally you can run:

```
BROWSE_SITE_LABEL='dev.arxiv.org' FLASK_APP=app.py FLASK_DEBUG=1 pipenv run flask run
```

I tried messing with the viewbox settings but couldn't get the exclamation icon smaller:
<img width="574" alt="Screen Shot 2020-11-08 at 9 58 17 PM" src="https://user-images.githubusercontent.com/746253/98495610-89fe5d80-220d-11eb-96a9-0a26b4259283.png">